sdk/eventhub/azure-mgmt-eventhub/dev_requirements.txt

```diff
@@ -1,3 +1,5 @@
 -e ../../../tools/azure-sdk-tools
-aiohttp>=3.0; python_version >= '3.5'
+../../core/azure-core
+../azure-eventhub
+../../tables/azure-data-tables
```

Comment:
we shouldn't be touching this file at all since this is actually a management plane library. We on the data plane team are not in charge of this library. Let's undo these changes

Reply:
ok

Reply:
so undoing the changes would mean adding this line again:
`aiohttp>=3.0; python_version >= '3.5'`

and removing these:
```
../../core/azure-core
../azure-eventhub
../../tables/azure-data-tables
```